src/renderer/components/+apps-helm-charts/helm-charts.tsx

```diff
@@ -24,13 +24,13 @@ import "./helm-charts.scss";
 import React, { Component } from "react";
 import type { RouteComponentProps } from "react-router";
 import { observer } from "mobx-react";
-import { helmChartsURL, IHelmChartsRouteParams } from "./helm-charts.route";
 import { helmChartStore } from "./helm-chart.store";
 import type { HelmChart } from "../../api/endpoints/helm-charts.api";
 import { HelmChartDetails } from "./helm-chart-details";
 import { navigation } from "../../navigation";
 import { ItemListLayout } from "../item-object-list/item-list-layout";
 import { SearchInputUrl } from "../input";
+import { helmChartsURL, IHelmChartsRouteParams } from "../../../common/routes";
```

Comment:
[nit] Maybe this? And, did you forgot to remove the `I` from `IHelmChartsRouteParams`?
```suggestion
import { helmChartsURL } from "../../../common/routes";
import type { IHelmChartsRouteParams } from "../../../common/routes";
```

Reply:
I did thanks.